packages/plugins/java/java/src/visitor.ts

```diff
@@ -62,7 +62,7 @@ export class JavaResolversVisitor extends BaseVisitor<JavaResolversPluginRawConf
       allImports.push(`java.util.stream.Collectors`);
     }
 
-    return allImports.map(i => `import ${i};`).join('\n') + '\n';
+    return allImports.map((i) => `import ${i};`).join('\n') + '\n';
```

Comment:
In this and the other arrow function updates, I didn't explicitly add the parens around the variables - not sure if this was done by a linter on the project when I committed or if it was done by some other automatic process. Let me know if I should change these back or leave them.